### PR TITLE
feature/blog and bugfix/banner

### PR DESCRIPTION
**Pull request by MichalCichon1**
styling the blog section, changing the div to section in the banner, adding a container, adding a gap between the banner and the rest of the website

### CODE REVIEW

**Review by DagmaraSzadkowska — CHANGES_REQUESTED** [review]
main.min.css ;)

**Comment by MichalCichon1** [comment]
chyba done :P

**Review by DagmaraSzadkowska — APPROVED** [review]

File: src/partials/banner.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="banner" id="banner">
+<section class="banner container" id="banner">
```
Comment (ArletaJanysz) [inline src/partials/banner.html:1]: section class="banner section"

File: src/partials/blog.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<section class="blog section" id="blog">
-  <div class="container">
-    <h2 class="section__title">BLOG</h2>
+<section class="blog section container" id="blog">
```
Comment (ArletaJanysz) [inline src/partials/blog.html:1]: bez container

File: src/sass/partials/_blog.scss [hunk 3]
```diff
@@ -1,45 +1,83 @@
...
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  &--prev {
-    width: 50px;
-    height: 50px;
-    @include tablet {
-      width: 61px;
-      height: 61px;
+@media screen and (min-width: 1200px) {
```
Comment (ArletaJanysz) [inline src/sass/partials/_blog.scss:67]: @media... do usunięcia

**Comment by MichalCichon1** [comment]
zrobione

**Review by DagmaraSzadkowska — APPROVED** [review]

**Pull request merged**